pkg/monitortests/kubeapiserver/auditloganalyzer/monitortest.go

```diff
@@ -255,6 +256,28 @@ func (w *auditLogAnalyzer) EvaluateTestsFromConstructedIntervals(ctx context.Con
 
 	}
 
+	for resource, numberOfApplies := range w.excessiveApplyChecker.resourcesToNumberOfApplies {
+		if numberOfApplies < 200 {
+			continue
```

Comment:
A test must always produce a result if it ran, so if you're doing a test per resource, it must return a success junit, otherwise all the stats in sippy and component readiness will go sideways and think it always flakes/fails.

With that in mind, how many tests are we talking here when that is corrected? It would be a massive number wouldn't it? If so, this may have to become a more generic single test.

Reply:
Yes, it would add hundred of tests if we create a test for each resource type. Since resource is already part of the error message, I'll create one test to rule them all